collect_app/src/main/java/org/odk/collect/android/activities/DeleteSavedFormActivity.java

```diff
@@ -18,18 +18,30 @@
 import android.os.Bundle;
 
 import androidx.appcompat.widget.Toolbar;
+import androidx.lifecycle.ViewModelProvider;
 import androidx.viewpager2.widget.ViewPager2;
 
 import com.google.android.material.tabs.TabLayout;
 import com.google.android.material.tabs.TabLayoutMediator;
 
 import org.odk.collect.android.R;
-import org.odk.collect.android.adapters.FileManagerTabsAdapter;
+import org.odk.collect.android.adapters.DeleteFormsTabsAdapter;
+import org.odk.collect.android.formmanagement.BlankFormsListViewModel;
+import org.odk.collect.android.injection.DaggerUtils;
+
+import javax.inject.Inject;
 
 public class DeleteSavedFormActivity extends CollectAbstractActivity {
+
+    @Inject
+    BlankFormsListViewModel.Factory viewModelFactory;
+    BlankFormsListViewModel viewModel;
```

Comment:
This might seem a little weird but my hope is that the `BlankFormsListViewModel` can be shared between Fill Blank Form and Delete Blank Form